src/Winfile.vcxproj.filters

```diff
@@ -102,6 +102,15 @@
     </None>
     <None Include="lang\res_pl-PL.rc">
       <Filter>lang</Filter>
+    <None Include="lang\suggest_he-IL.db">
```

Comment:
The <None></None> got mismatched.  The stray </None> from below needs to be moved up here.